### ISSUE TITLE
Assignment of invite roles currently not working

### ISSUE THREAD

**Issue by TheLlamamarketer** [Contraversum/ContraBot#67]
It seems that the feature that worked on the test server has some sort of problems on the actual server.

**Comment by johan-t** [comment]
Can you check if all of the ids are correct for the roles?

**Comment by johan-t** [comment]
But we would need to fix this to work on issue #66 which is important to get some viral growth

**Comment by TheLlamamarketer** [comment]
I already checked that the roles were correct, maybe the condition that when a new user is invited never really worked, that being the main issue?

**Comment by johan-t** [comment]
The problem seems to with the missing privileged Gateway Intent

**Comment by johan-t** [comment]
Yeah we forgot to enable it for the production bot, we only enabled them on the test bot
